src/python/CRABClient/CrabRestInterface.py

```diff
@@ -0,0 +1,244 @@
+"""
+Handles client interactions with remote REST interface
+"""
+
+from __future__ import division
+from __future__ import print_function
+
+import os
+import time
+import random
+import subprocess
+from ServerUtilities import encodeRequest
+import json
+
+try:
+    from urllib import quote as urllibQuote  # Python 2.X
+except ImportError:
+    from urllib.parse import quote as urllibQuote  # Python 3+
+
+import logging
+
+try:
+    from TaskWorker import __version__
+except:  # pylint: disable=bare-except
+    try:
+        from CRABClient import __version__
+    except:  # pylint: disable=bare-except
+        __version__ = '0.0.0'
+
+EnvironmentException = Exception
+
+def retriableError(http_code, exitcode):
+    """ Return True if the error can be retried
```

Comment:
indicate in the comments what the argumets are: http_code is the code form the http call if it was possible to do the call and get it, otherwise exitcode is the exit code of the curl command that was forked to execute the call. All in all this funciont is now specific of using curl to make the call. Maybe rename exitcode to curlExitCode ?